master/buildbot/reporters/github.py

```diff
@@ -205,5 +216,6 @@ def createStatus(self,
         payload = {'body': description}
 
         return self._http.post(
-            '/'.join(['/repos', repo_user, repo_name, 'issues', issue, 'comments']),
+            '/'.join(['/repos', repo_user, repo_name,
```

Comment:
Ya I just tried again with --max-line-length 132
It was still reformatting this line. I can manually set it back